app/templates/password/reset.hbs

```diff
@@ -0,0 +1,3 @@
+{{password/reset-password 
+  resetPassword=(route-action "resetPassword")
```

Comment:
This is at a route level, so no need to generate a controller just for the sake of not using `route-action`, I think. The component is just as easily testable this way as it would be with a controller, but the boilerplate is reduced.

Also, thank you for that input, @snewcomer. I was in a mental place where I wasn't sure how route actions are useful exactly, since, in my experience, using them in nested components makes everything harder to test.

This example is where using a route action is useful for reducing boilerplate with no (to me) apparent negatives.